assets/scss/layout.scss

```diff
@@ -1,13 +1,3 @@
-.Site {
-  background-color: hsl(52, 1%, 92%);
-  display: flex;
-  min-height: 100vh;
-  flex-direction: column;
-}
-
-#main-nav {
-  a {
-    color: white;
-    font-weight: bold;
-  }
+.topNav {
+  border-top: none;
```

Comment:
`border-top: 0` is preferred over `border-top: none`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#borderzero)